src/services/request.service.js

```diff
@@ -27,4 +28,27 @@ export default class RequestService {
       }
     );
   }
+
+  /**
+   * @returns {Integer} returns number of traveled places
+   * @description This function getMostTraveledDestinations()
+   *  returns all of the destinations a and their visit times
+   */
+  static getMostTraveledDestinations = async () => {
+    const placeAndTheirVisitTimes = await sequelize.query('SELECT destination, COUNT(destination) FROM requests WHERE status !=\'pending\' AND status!=\'rejected\' GROUP BY destination ORDER BY count DESC');
+    
+    return placeAndTheirVisitTimes;
+  }
+
+
+   /**
+   * @param {Integer} id number of traveled places
+   * @returns {Integer} returns number of traveled places
+   * @description This function getMostTraveledDestinations()
```

Comment:
Is this description accurate?

Reply:
I changed it, made it short but descriptive enough

Reply:
I still can't see the change in its description

Reply:
@karamuka I changed it, if you go in codes, you can find it, I don't know why isn't appearing here on comments
![Screenshot from 2020-03-23 15-48-07](https://user-images.githubusercontent.com/56886064/77323349-c97b4680-6d1d-11ea-9026-d45bf98a54b6.png)